org.civicrm.reqangular/test/services/format-currency.spec.js

```diff
@@ -30,27 +30,32 @@ define([
     var testMap = [
       // [thousandSeparator, decimalSeparator, inputAmount, outputAmount]
       // Common cases
+      [',', '*', 0.0, '0'],
+      [',', '*', 'xxxxx', '0'],
+      [',', '*', '0.8889', '0*88'],
+      [',', '*', '-0.8889', '0*88'],
```

Comment:
Is this confirmed that negative currency goes positive?

Reply:
yes! confirmed with @mike